.github/workflows/create-release.yml

```diff
@@ -37,10 +37,21 @@ jobs:
         pack build testapp -p samples/go/mod --builder testbuilder
         pack build testapp -p samples/java/maven --builder testbuilder
 
+  check-builder-changes:
+    name: Check For Builder Changes
+    runs-on: ubuntu-latest
+    needs: smoke
+    steps:
+    - name: Compare With Previous Commit
+      uses: actions/checkout/@v2
+      with:
+        fetch-depth: 0 # gets full history
+    - run: if [ -z "$(git diff $(git describe --tags --abbrev=0) -- builder.toml)" ]; then exit 1; else exit 0; fi
+
```

Comment:
Won't this create a red/failing job every time there is a commit to the repo that doesn't change `builder.toml`?

Maybe we want to set an env var and use that to conditionally release, instead of the exit code?

https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#about-contexts-and-expressions

Reply:
I've switched things up to use a step output + conditional. This has the benefit of not failing a job, but the drawback is that since jobs can't easily share information afaik, we have to add the conditional to all of the steps that follow the check for builder changes.